python_modules/dagster/dagster/core/snap/execution_plan_snapshot.py

```diff
@@ -63,7 +62,6 @@ def __new__(
         step_keys_to_execute: Optional[List[str]] = None,
```

Comment:
this is the part most likely to break things since we persist these snapshots - but if they're not longer required to be included then we should be good? Maybe add to the 'serdes log' comments just above